context-extraction/src/main/java/de/atb/context/persistence/context/ContextRepository.java

```diff
@@ -71,11 +70,12 @@ private ContextRepository() {
     }
 
     private synchronized void persistNamedContext(ContextContainer context) {
-        Store store = getStore(context.getBusinessCase());
-        Model named = SDBFactory.connectNamedModel(store, context.getIdentifier());
+// FIXME to fix
```

Comment:
This method's body contains only commented out code.
Is there still something to be fixed or can it be removed completely?